### ISSUE TITLE
Image of connections

### ISSUE THREAD

**Issue by lcava000** [hideakitai/ArtNet#28]
Thank you for your contribution.
But can you post an image of the connections with ESP32 and RS485 to the DMX? Thank you

**Comment by hideakitai** [comment]
You can't use ArtNet with RS485. Maybe related: https://arcattack.com/dmx-over-rs485-with-an-arduino

**Comment by lcava000** [comment]
I would like to use ESP32 as an ART-NET receiver.
I connected to the esp32 according to these links.

  - RS-485 -
DI -> TX2 UART
DE -> GND (Receive Mode)
RE -> GND (Receive Mode)
RE -> RX2 UART
VCC -> VCC
GND -> GND
A -> DATA + DMX
B -> DATA - DMX

Why i can't use ArtNet with RS485?
Below I found a wiring diagram for arduino. 

![image](https://user-images.githubusercontent.com/81354169/146521292-b7bbaeb1-dd29-4b5d-95ff-1f0d2e2a3956.png)
